.golangci.yml

```diff
@@ -1,3 +1,12 @@
 linters:
   enable:
-    - revive
+    - errorlint
```

Comment:
There is a list of linters [enabled by default](https://golangci-lint.run/usage/linters/), and others (like those) can be added manually